.github/workflows/terraform-create-git-tag.yml

```diff
@@ -12,22 +12,26 @@ on:
         type: string
 
 jobs:
-  create_tag:
+  build:
+    if: github.event.pull_request.merged == true
     runs-on: ubuntu-latest
-
-    env:
-      GIT_USER_NAME: "Infra Workleap"
-      GIT_USER_EMAIL: "infra@workleap.com"
-      SOURCE_VERSION_MESSAGE: "Automated tag creation"
-
+    permissions:
+      contents: write
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.merge_commit_sha }}
+        fetch-depth: '0'
+
+    - name: Set git author
+      run: |
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "github-actions[bot]@users.noreply.github.com"
 
-      - name: Create Git Tag
-        run: |
-          echo "Creating tag [${buildVersion}] with message: ${{ env.SOURCE_VERSION_MESSAGE }}"
-          git config --global user.name "${{ env.GIT_USER_NAME }}"
-          git config --global user.email "${{ env.GIT_USER_EMAIL }}"
-          git tag -a ${buildVersion} -m "${{ env.SOURCE_VERSION_MESSAGE }}"
-          git push origin --tags
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@v1 # Don't use @master or @v1 unless you're happy to test the latest version
```

Comment:
What's up with this comment? Should we use a sha digest instead? Or a more specific version like the current @v1.71.0 (is it even possible? I never tried a full tag before)?

Should Renovate update this non-official/well-known/trusted action? Two years ago, our security department asked to pin the digest of actions. I don't know if it still applies today.

Reply:
Sorry I copy/pasted this comment from the upstream page of the github action. I will go perform these changes in a new PR.